Paintroid/src/main/res/drawable/circle.xml

```diff
@@ -17,7 +17,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->
 <shape android:shape="oval" xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@color/dialog_subtitle"/>
+    <solid android:color="@color/color_chooser_black"/>
```

Comment:
Please use a different color, e.g. `@android:color/black`
`color_chooser_black` should only be used in the color picker.